resources/i18n/en.json

```diff
@@ -855,10 +855,23 @@
     "ROCMGPUsupport": "ROCm GPU support",
     "DescROCMGPUsupport": "AMD ROCm GPU support. <br/>Requires Backend.AI ROCm Plugin.",
     "Scheduler": "Scheduler",
-    "JobScheduler": "Choose default job scheduler",
-    "JobSchedulerUpdated": "Default job scheduler has been updated.",
+    "SchedulerConfiguration": "Configuration per job scheduler.",
     "Require1912orAbove": "Requires Backend.AI 19.12 or above.",
     "Require2003orAbove": "Requires Backend.AI 20.03 or above.",
+    "Config": "Config",
+    "JobSchedulerUpdated": "Default job scheduler has been updated.",
+    "ConfigPerJobSchduler": "Configuration<br />per job scheduler",
```

Comment:
Also, the title needs to imply what is this dialog will exactly do.
In my opinion `default scheduler` is kind of a vague explanation, therefore I changed the title and description to `Configuration per job scheduler` for better explanation.